SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -108,91 +107,26 @@ void SetLevelEditorPlaySettingsWorkerType(const FWorkerTypeLaunchSection& InWork
 uint32 GetWorkerCountFromWorldSettings(const UWorld& World)
 {
 	const ASpatialWorldSettings* WorldSettings = Cast<ASpatialWorldSettings>(World.GetWorldSettings());
-
 	if (WorldSettings == nullptr)
 	{
 		UE_LOG(LogSpatialGDKDefaultLaunchConfigGenerator, Error, TEXT("Missing SpatialWorldSettings on map %s"), *World.GetMapName());
 		return 1;
 	}
 
-	if (WorldSettings->IsMultiWorkerEnabled() == false)
-	{
-		return 1;
-	}
-
-	FSpatialGDKEditorModule& EditorModule = FModuleManager::GetModuleChecked<FSpatialGDKEditorModule>("SpatialGDKEditor");
-	uint32 NumWorkers = 0;
-	if (WorldSettings->DefaultLayerLoadBalanceStrategy == nullptr)
+	if (!WorldSettings->IsMultiWorkerEnabled())
 	{
-		UE_LOG(LogSpatialGDKDefaultLaunchConfigGenerator, Error, TEXT("Missing Load balancing strategy on map %s"), *World.GetMapName());
 		return 1;
 	}
-	else
-	{
-		UAbstractRuntimeLoadBalancingStrategy* LoadBalancingStrat = nullptr;
-		FIntPoint Dimension;
-		if (!EditorModule.GetLBStrategyExtensionManager().GetDefaultLaunchConfiguration(WorldSettings->DefaultLayerLoadBalanceStrategy->GetDefaultObject<UAbstractLBStrategy>(), LoadBalancingStrat, Dimension))
-		{
-			UE_LOG(LogSpatialGDKDefaultLaunchConfigGenerator, Error, TEXT("Could not get the default SpatialOS Load balancing strategy from %s"), *WorldSettings->DefaultLayerLoadBalanceStrategy->GetName());
-			NumWorkers += 1;
-		}
-		else
-		{
-			NumWorkers += LoadBalancingStrat->GetNumberOfWorkersForPIE();
-		}
-	}
-
-	for (const auto& Layer : WorldSettings->WorkerLayers)
-	{
-		const FName& LayerKey = Layer.Key;
-		const FLayerInfo& LayerInfo = Layer.Value;
-
-		UAbstractRuntimeLoadBalancingStrategy* LoadBalancingStrat = nullptr;
-		FIntPoint Dimension;
-		if (LayerInfo.LoadBalanceStrategy == nullptr)
-		{
-			UE_LOG(LogSpatialGDKDefaultLaunchConfigGenerator, Error, TEXT("Missing Load balancing strategy on layer %s"), *LayerKey.ToString());
-			NumWorkers += 1;
-		}
-		else if (!EditorModule.GetLBStrategyExtensionManager().GetDefaultLaunchConfiguration(LayerInfo.LoadBalanceStrategy->GetDefaultObject<UAbstractLBStrategy>(), LoadBalancingStrat, Dimension))
-		{
-			UE_LOG(LogSpatialGDKDefaultLaunchConfigGenerator, Error, TEXT("Could not get the SpatialOS Load balancing strategy for layer %s"), *LayerKey.ToString());
-			NumWorkers += 1;
-		}
-		else
-		{
-			NumWorkers += LoadBalancingStrat->GetNumberOfWorkersForPIE();
```

Comment:
I see we no longer call into `GetNumberOfWorkersForPIE`. Is this functionality intentionally replaced with GetMinWorkers? If so should we remove it from `UAbstractRuntimeLoadBalancingStrategy` ?

Reply:
why remove it from the file when I could just delete the whole thing, good idea